### PR TITLE
Qa9 115.error registering with existing email

### PR DESCRIPTION
**Pull request by lakicandjela**

### CODE REVIEW

File: e2e/tests.spec.ts [hunk 1]
```diff
@@ -0,0 +1,62 @@
+import {test, expect} from "@playwright/test"
+import { PageManager } from "../page-objects/pageManager"
+import {faker} from '@faker-js/faker'
```
Comment (testinginternship) [inline e2e/tests.spec.ts:3]: Do we need to import something that is not used?

File: e2e/tests.spec.ts [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+    // Verify the cart is empty
+    await expect(page.getByText('You have no items in your')).toBeVisible()
+
+})
+
+test('Error message is there while registering with existing email', async({page}) => {
+    // Setup:
+    const pm = new PageManager(page); // Create a PageManager for page interactions
+    await pm.navigateTo().createAccountPage(); // Go to the account creation page
```
Comment (testinginternship) [inline e2e/tests.spec.ts:44]: Please add assertions to each step in the test! Thanks

Comment (testinginternship) [reply e2e/tests.spec.ts:44]: Good job with comments!

File: page-objects/checkoutPage.ts [hunk 3]
```diff
@@ -1,13 +1,31 @@
 import {Locator, Page} from '@playwright/test'
```
Comment (testinginternship) [inline page-objects/checkoutPage.ts:1]: Do we need to import something that is not used?

File: page-objects/checkoutPage.ts [hunk 4]
```diff
@@ -1,13 +1,31 @@
...
+     * @param {string} streetAddress2 - Street Address Line 2 (optional).
+     * @param {string} streetAddress3 - Street Address Line 3 (optional).
+     * @param {string} city - The billing address city.
+     * @param {string} zip - The billing address zip/postal code.
+     * @param {string} country - The billing address country.
+     * @param {string} phoneNumber - The billing address phone number.
+     * @param {string} stateProvince - The billing address state/province.
+     * @param {ShippingRate} shippingRate - An enum value specifying the shipping rate type (e.g., Fixed, Table Rate).
+     */
     async checkoutWhileNotSignedIn(email: string, firstName: string, lastName: string, company: string, streetAddress1: string, streetAddress2: string,
```
Comment (testinginternship) [inline page-objects/checkoutPage.ts:29]: Do we need to use an implicit wait In checkoutWhileNotSignedIn or is there another way?

File: page-objects/checkoutPage.ts [hunk 5]
```diff
@@ -26,11 +44,14 @@ export class CheckoutPage extends HelperBase{
...
+            // State/Province is a required dropdown
             await this.page.locator('select[name="region_id"]').selectOption(stateProvince)
         } else if(classAtr=='field'){
+            // State/Province is a free-form text field
             await this.page.getByRole('textbox', {name: 'State/Province'}).fill(stateProvince)
         }
 
+        // Select Shipping Rate
         if (shippingRate as string === ShippingRate.FIXED) {
             await this.page.getByLabel('Fixed').check();
```
Comment (testinginternship) [inline page-objects/checkoutPage.ts:56]: Please add assertions!

File: page-objects/checkoutPage.ts (hunk 4 above)
Comment (lakicandjela) [reply page-objects/checkoutPage.ts:29]: I haven't used this function for my tests so far (I made it while we were preparing the project structure). So if it is okay, if there is a need to use, it I will correct this.

File: e2e/tests.spec.ts (hunk 2 above)
Comment (lakicandjela) [reply e2e/tests.spec.ts:44]: Thanks!

**Pull request merged**